scripts/configs/behavior_pick_place_evaluation.yaml

```diff
@@ -34,8 +34,7 @@ APPROACHES:
   # of the below GNN approaches.
   # gnn_shooting:  # Model-based GNN option policy baseline.
   #   NAME: "gnn_option_policy"
-  # NOTE: this needs to run after Model-based GNN has already been run.
```

Comment:
Is this not still true?

Reply:
nope - not true! in fact it was never true lol - we were lying to ourselves because of something in our old scripts.